### PR TITLE
[896][IMP] Add mail_outbound_static as dependency

### PR DESCRIPTION
**Pull request by TimLai125**
#[896](https://www.quartile.co/web?debug=#id=896&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by TimLai125** [comment]
The branch to be rebased after https://github.com/qrtl/sst-custom/pull/287 is maerged

**Review by yostashiro — APPROVED** [review]
LGTM, but please also add applicable changes from https://github.com/qrtl/sst-custom/pull/289.

File: website_forum_adj/README.rst [hunk 1]
```diff
@@ -10,18 +10,19 @@ Website Forum Adjustment
...
     :alt: Beta
-.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
-    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-    :alt: License: AGPL-3
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-qrtl%2Fsst--custom-lightgray.png?logo=github
-    :target: https://github.com/qrtl/sst-custom/tree/11.0/website_forum_adj
+    :target: https://github.com/qrtl/sst-custom/tree/12.0/website_forum_adj
```
Comment (yostashiro) [inline website_forum_adj/README.rst]: Why 12.0?

File: website_forum_adj/README.rst [hunk 2]
```diff
@@ -49,6 +50,6 @@ Authors
 Maintainers
 ~~~~~~~~~~~
 
-This module is part of the `qrtl/sst-custom <https://github.com/qrtl/sst-custom/tree/11.0/website_forum_adj>`_ project on GitHub.
+This module is part of the `qrtl/sst-custom <https://github.com/qrtl/sst-custom/tree/12.0/website_forum_adj>`_ project on GitHub.
```
Comment (yostashiro) [inline website_forum_adj/README.rst]: Why 12.0?

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Comment by TimLai125** [comment]
Drop this PR since the logic will be generalized in https://github.com/qrtl/sst-custom/pull/289

**Pull request closed without merging**